airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -91,7 +97,7 @@ def wait_for_pipeline_state(
         namespace: str = "default",
         success_states: list[str] | None = None,
         failure_states: list[str] | None = None,
-        timeout: int = 5 * 60,
+        timeout: int = 7 * 60,
```

Comment:
Any reason why it's now 7 minutes and not 5?

Reply:
I faced some problem with waiting for the pipeline state and thought that increasing the timeout will solve the problem.  I will reduce the timeout as it was :)

Reply:
@potiuk Thank you for the approve!
I have update the file